Classes/Systems/ShortcutHandler.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  ShortcutHandler.swift
+//  Freetime
+//
+//  Created by Viktor Gardart on 2017-10-08.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+
+struct ShortcutHandler {
+
+    private enum Items: String {
+        case search
+        case switchAccount
+    }
+
+    static func configure(application: UIApplication, sessionManager: GithubSessionManager) {
+        application.shortcutItems = generateItems(sessionManager: sessionManager)
+    }
+
+    static func handle(shortcutItem item: UIApplicationShortcutItem,
+                       sessionManager: GithubSessionManager,
+                       navigationManager: RootNavigationManager) -> Bool {
+        let itemType = Items(rawValue: item.type)
+        switch itemType {
+        case .some(.search):
+            navigationManager.selectViewController(atIndex: 1)
+            break
+        case .some(.switchAccount):
+            if let index = item.userInfo?["sessionIndex"] as? Int {
+                let session = sessionManager.userSessions[index]
+                sessionManager.focus(session, dismiss: false)
+            }
+            break
+        default:
+            return false
+        }
+        return true
```

Comment:
I feel this is not as readable as it could be. Would prefer two `return`s instead of two `break`s. Also, do we silently want to ignore the `default` case?

Reply:
Can fix the two `return`, 
About the `default` case: that wont probably happen and if it does, what should we do? Since we add the `Items` in `generateItems` we have full control over what actions will be sent.

Reply:
It's a tricky one, yeah - `fatalError`ing would probably not be what we want to do. Let's leave it as is for now 👍

Reply:
Once we get the `toaster` in place we could probably do something with that.